app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More about teachers' salaries and benefits</h2>
+
+    <div class="inset">
+      <p>
+        Your teaching salary will depend on your role in the school and qualifications.
+        There are lots of opportunities to increase your salary during your career:
+      </p>
+
+      <ul>
+        <li>teachers who make a sustained contribution to their school could expect to earn £40k to £43k or more in London</li>
```

Comment:
```suggestion
        <li>teachers who make a sustained contribution to their school could expect to earn over £40k</li>
```
@philspray Is it worth simplifying this? I think it is slightly misleading as it sounds like everyone in London on upper pay gets over 43k, which they don't